k8s/cmd/crossconnect-monitor/crossconnect_monitor.go

```diff
@@ -64,14 +63,9 @@ func monitorCrossConnects(address string, continuousMonitor bool) {
 
 		prom, err := tools.ReadEnvBool(metricspkg.PrometheusEnv, metricspkg.PrometheusDefault)
 		if err == nil && prom && event.Type == crossconnect.CrossConnectEventType_UPDATE {
-			metrics, metricsIdentifiers, errm := getMetrics(event)
-			if errm != nil {
-				logrus.Infof("failed to get metrics: %v", err)
-			} else {
-				servePrometheus(metricsIdentifiers, metrics)
-			}
+			trackMetrics(event)
 		} else {
-			logrus.Infof("failed to serve prometheus: env PROMETHEUS=%t, err: %v", prom, err)
+			logrus.Warningf("failed to serve prometheus: env PROMETHEUS=%t, err: %v", prom, err)
```

Comment:
This warning is sent even if the event is not an UPDATE event

Reply:
Thanks @matrohon